test/e2e/node/pods.go

```diff
@@ -147,7 +114,7 @@ var _ = SIGDescribe("Pods Extended", func() {
 					return false, nil
 				}
 				for _, kubeletPod := range podList.Items {
-					if pod.Name != kubeletPod.Name {
+					if pod.Name != kubeletPod.Name || pod.Namespace != kubeletPod.Namespace {
```

Comment:
And uid?

Reply:
after adding namespace, I realized the pod name contains a random uuid, so probably not necessary?